nodes/usb_cam_node.cpp

```diff
@@ -414,6 +417,8 @@ class UsbCamNode
 int main(int argc, char **argv)
 {
   ros::init(argc, argv, "usb_cam");
+  ros::AsyncSpinner spinner{ 2 };
```

Comment:
Add some kind of note here that if we add another callback type without adding another thread, there could be weird threading issues